pkgs/tools/text/html2text/default.nix

```diff
@@ -1,35 +1,24 @@
-{ lib, stdenv, fetchurl }:
+{ lib, stdenv, fetchFromGitLab, autoreconfHook, libiconv }:
 
 stdenv.mkDerivation rec {
   pname = "html2text";
-  version = "1.3.2a";
+  version = "2.2.3";
 
-  src = fetchurl {
-    url = "http://www.mbayer.de/html2text/downloads/html2text-${version}.tar.gz";
-    sha256 = "000b39d5d910b867ff7e087177b470a1e26e2819920dcffd5991c33f6d480392";
+  src = fetchFromGitLab {
+    owner = "grobian";
+    repo = "html2text";
+    rev = "40c175152835fc41f01d89058145c3da4f62ae3a";
```

Comment:
```suggestion
    rev = "v${version}";
```

Reply:
Thanks, done! (I force pushed to keep the commit history neat).

Probably you already did, but I verified the tagged commit still pointed to that revision, so the hash is the same.